src/modules/new-request-form/fields/attachments/Attachments.tsx

```diff
@@ -61,8 +61,20 @@ export function Attachments({ field }: AttachmentProps): JSX.Element {
   const { addToast } = useToast();
   const { t } = useTranslation();
 
+  const errorMessage = (file: File, xhr: XMLHttpRequest) => {
+    if (xhr.response?.error == "RecordInvalid") {
```

Comment:
Note to self: also check that `xhr.response?.details?.base` is present here because there may be an InvalidRecord response without that key in the details